src/components/spreadsheet/utils/config-tables.js

```diff
@@ -622,32 +958,237 @@ export const TABLES_DEFINITIONS = {
+                cellStyle: (params) =>
+                    isEditableTwtPhaseRegulatingTerminalCell(params)
+                        ? editableCellStyle(params)
+                        : {},
+                editable: isEditableTwtPhaseRegulatingTerminalCell,
+                crossValidation: {
+                    requiredOn: {
+                        dependencyColumn: 'phaseTapChanger.regulationType',
+                        columnValue: REGULATION_TYPES.DISTANT.id,
+                    },
+                },
+                cellEditor: TWTPhaseRegulatingTerminalEditor,
+                cellEditorParams: (params) => {
+                    return {
+                        defaultValue: (params) => {
+                            getTapChangerRegulationTerminalGeneratorValue(
+                                params
+                            );
+                        },
+                        gridContext: params.context,
+                        gridApi: params.api,
+                        colDef: params.colDef,
+                        rowData: params.data,
+                    };
+                },
+                cellEditorPopup: true,
+            },
+            {
+                id: 'PhaseLowTapPosition',
+                field: 'phaseTapChanger.lowTapPosition',
+                getQuickFilterText: excludeFromGlobalFilter,
+                filter: 'agNumberColumnFilter',
+                numeric: true,
+                fractionDigits: 0,
+                editable: (params) =>
+                    isEditable(params) &&
+                    params.data?.phaseTapChanger?.steps?.length > 0,
+                cellStyle: editableCellStyle,
+                cellEditor: 'agSelectCellEditor',
+                cellEditorParams: (params) => {
+                    return {
+                        values: generateTapPositions(
+                            params.data?.phaseTapChanger
+                        ),
+                    };
+                },
+                valueSetter: (params) => {
+                    params.data.phaseTapChanger = {
+                        ...(params.data?.phaseTapChanger || {}),
+                        lowTapPosition: params.newValue,
+                    };
+                    return params;
+                },
+                crossValidation: {
+                    requiredOn: {
+                        dependencyColumn: 'phaseTapChanger.steps',
+                    },
+                },
+            },
+            {
+                id: 'PhaseHighTapPosition',
+                field: 'phaseTapChanger.highTapPosition',
+                valueGetter: (params) =>
+                    computeHighTapPosition(
+                        params?.data?.phaseTapChanger?.steps
+                    ),
                 getQuickFilterText: excludeFromGlobalFilter,
             },
             {
                 id: 'PhaseTap',
                 field: 'phaseTapChanger',
```

Comment:
```suggestion
                field: 'phaseTapChanger.tapPosition',
```
I spotted some validation bugs without this change : the tap position of the phase tap changer was set to error after modifying unrelated fields